test/e2e/aadpodidentity_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package aadpodidentity
```

Comment:
Can you please add a README with the various steps and files/code involved. Short one is fine but which could make it easy for a new developer to understand what is goin going on and what are the steps needed to add the new tests etc...